### DISCUSSION TITLE
Player Indicators: In Combat-Bracket Only

### DISCUSSION THREAD

**Discussion by ETalanian** [runelite/runelite#17894] [Ideas]
Please include an option on Player Indicators while in wildy to only show those within your level bracket.

If this is too annoying to code, an option to set a level range in the plugin to only show those names.

Thank you <3

**Comment by cdfisher** [comment]
This would almost certainly be seen as falling afoul of the [third-party client guidelines](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1), specifically that it would

> Offers additional information about other players, with the purpose of scouting PvP targets

**Reply by ETalanian** [reply]
Damn, that's unfortunate.  I'm just sick of morons trying to pk me at black chins.
